### PR TITLE
[RAPPS] Show listview context menu when the keyboard menu button/Shift+F10 is pressed

### PR DESCRIPTION
**Pull request by whindsaks**
NM_RCLICK is too low level, WM_CONTEXTMENU is the correct place to handle context menus.

### CODE REVIEW

File: base/applications/rapps/appview.cpp [hunk 1]
```diff
@@ -1525,6 +1516,30 @@ CApplicationView::ProcessWindowMessage(
...
+                        pt.x = r.left + (r.right - r.left) / 2;
+                        pt.y = r.top + (r.bottom - r.top) / 2;
+                        ::ClientToScreen((HWND)wParam, ppt = &pt);
+                    }
+                    ShowPopupMenuEx(m_hWnd, m_hWnd, 0, ID_INSTALL, ppt);
+                    return TRUE;
+                }
+            }
+        }
+        break;
```
Comment (binarymaster) [inline base/applications/rapps/appview.cpp:1542]: It looks quite more complicated than it was previously, is it really worth it?

Comment (whindsaks) [reply base/applications/rapps/appview.cpp:1542]: I'd argue that yes, you are supposed to be able to access the context menu with the keyboard even if the commands are available on the toolbar.

It is not that complicated, 3 extra function calls.

**Comment by whindsaks** [comment]
> This PR is stale because it received no updates in the last 4 months. Without removing the stale label, or commenting on this ticket it will be closed in 2 weeks.

Go away bot

**Comment by whindsaks** [comment]
No bot!

File: base/applications/rapps/appview.cpp [hunk 2]
```diff
@@ -1525,6 +1516,30 @@ CApplicationView::ProcessWindowMessage(
             OnCommand(wParam, lParam);
         }
         break;
+
+        case WM_CONTEXTMENU:
+        {
+            bool kbd = -1 == (int)(INT_PTR)lParam;
+            if ((HWND)wParam == m_ListView->m_hWnd)
+            {
+                int item = ListView_GetNextItem((HWND)wParam, -1, LVNI_FOCUSED | LVNI_SELECTED);
```
Comment (learn-more) [inline base/applications/rapps/appview.cpp:1525]: You could just do the stuff like `m_ListView.GetNextItem`

**Review by learn-more — APPROVED** [review]

**Pull request merged**